source/MaterialXContrib/MaterialXNode/resources/tiledImage.xml

```diff
@@ -21,7 +21,7 @@
     <int name="tiled_image3_frameendaction" value="1" />
   </values>
   <outputs>
-    <float3 name="tiled_image3_output" />
+    <float3 name="out" />
```

Comment:
Should not be needed now with the new output attr checking. Will try to use different name later.